src/algebra/lattice_ordered_group.lean

```diff
@@ -127,89 +115,62 @@ namespace lattice_ordered_comm_group
 
 -- Bourbaki A.VI.12 Definition 4
 /--
+Let `α` be a lattice ordered commutative group with identity `1`. For an element `a` of type `α`,
+the element `a ⊔ 1` is said to be the *positive component* of `a`, denoted `a⁺`.
+-/
+@[to_additive pos /-"
 Let `α` be a lattice ordered commutative group with identity `0`. For an element `a` of type `α`,
 the element `a ⊔ 0` is said to be the *positive component* of `a`, denoted `a⁺`.
--/
-@[to_additive pos
-  "Let `α` be a lattice ordered commutative group with identity `0`. For an element `a` of type `α`,
-  the element `a ⊔ 0` is said to be the *positive component* of `a`, denoted `a⁺`."]
-def mpos (a : α) : α :=  a ⊔ 1
-postfix `⁺`:1000 := mpos
+"-/,
+priority 100
+] -- see Note [lower instance priority]
+instance has_one_lattice_has_pos_part : has_pos_part (α)  := ⟨λa, a ⊔ 1⟩
```

Comment:
The additive name no longer really matches the multiplicative name. Is that intentional?

Reply:
Yes. You suggested this convention here: https://github.com/leanprover-community/mathlib/pull/8673#discussion_r693308231

Reply:
Sorry for not returning to this earlier. What I mean is that you now have `pos` as the additive counterpart of `has_one_lattice_has_pos_part`. I think `pos/mpos` is fine. But here you probably want `has_zero_lattice_has_pos_part` or something like that... Not simply `pos`.

Reply:
Ah, sorry - I see what you mean now. I misread the diff. I've removed the name overrides now - they were an artifact left over from earlier.